src/core/connection.c

```diff
@@ -1119,6 +1119,20 @@ QuicConnReplaceRetiredCids(
     return TRUE;
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
```

Comment:
```suggestion
_IRQL_requires_max_(DISPATCH_LEVEL)
```
Not that it really matters, but there is no reason to require passive.

Reply:
all the functions in timer_wheel.c require passive level. I guess that's incorrect too?

Reply:
Anything that is just running logic can be at dispatch. If it calls into other layers (i.e. the rest of the connection), then it might need to be at passive. But don't worry about going and changing all that right now.